glide.yaml

```diff
@@ -20,7 +20,7 @@ import:
 - package: github.com/kardianos/osext
   version: master
 - package: go.uber.org/thriftrw
-  version: ^v1.16.0
+  version: ^v1.23.0
```

Comment:
Lets make sure we bump thriftrw in EG and EGO as well. Want to keep them all in sync.

Reply:
ack